pkg/controller/hostpathprovisioner/scc.go

```diff
@@ -213,7 +214,7 @@ func (r *ReconcileHostPathProvisioner) checkSCCUsed() (bool, error) {
 			// not using SCCs
 			return false, nil
 		}
-		return false, err
+		return false, nil
```

Comment:
same q as above, am I missing something obvious? do we not care about SCCs anymore with CSI?

Reply:
Yeah this is the main issue I have.  @awels says that is `NoMatchError` is no longer returned and I think that is something we should dig into and find out exactly what changed and if this is either 1) a bug 2) we should be doing something differently or 3) the new normal

Reply:
Sounds like this https://github.com/kubernetes-sigs/controller-runtime/issues/2354#issuecomment-1566874090